### ISSUE TITLE
Saucy/indigo build error

### ISSUE THREAD

**Issue by scpeters** [ros-simulation/gazebo_ros_pkgs#175]
I'm trying to build on saucy/indigo but getting the following error. Any ideas?

```
  File "/home/scpeters/ws/gazebo_ros_pkgs/src/gazebo_ros_pkgs/gazebo_plugins/cfg/GazeboRosCamera.cfg", line 6, in <module>
    from driver_base.msg import SensorLevels
ImportError: No module named msg
Built target nav_msgs_generate_messages_py
Built target nav_msgs_generate_messages_lisp
make[2]: *** [/home/scpeters/ws/gazebo_ros_pkgs/devel/include/gazebo_plugins/GazeboRosCameraConfig.h] Error 1
```

**Comment by scpeters** [comment]
Cross-posted to [ros answers](http://answers.ros.org/question/149716/gazebo_ros_pkgs-build-error-on-saucyindigo/)

**Comment by scpeters** [comment]
fixed by #176
